### PR TITLE
Revamp Promotion section

### PR DESCRIPTION
**Pull request by jt15s**
Revamped the Promotion section as suggested by @panos by moving the Promotion Team's page and adding a new index page talking about things individuals can do to promote Haiku.

### CODE REVIEW

File: content/community/getting-involved/promotion/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
++++
+Promotion is a crucial part of ensuring Haiku continues to grow. Without spreading the word, we cannot gain new users and volunteers. 
+
+Whilst Haiku *does* have a [Promotion Team, which you can read more about here](/community/getting-involved/promotion/promotion-team/), you don't need to join the Team to promote the Project, as you will see below.
+
+## What can I do to promote Haiku?
+Letting people know that you use Haiku can help more people discover the Project and decide whether they want to try the OS out. You might want to also mention that our ISOs allow for people to try out the OS without installing it onto a hard disk. You could consider burning the [latest release of Haiku](/get-haiku/) to a USB or CD (make sure the medium is virus-free though) - and pass it around to those around you who may be interested.
+
+Talking about Haiku online, can really help as well. You could, for instance, retweet the [official Haiku Twitter](https://twitter.com/haikuOS) account posts. You could post screenshots/photos of your Haiku setup, or show the world what new creation you've been working on in Haiku - all these and more help spread the word.
```
Comment (panos) [inline content/community/getting-involved/promotion/_index.md:14]: `Talking about Haiku online, can really help as well.` No comma needed here.

`You could, for instance, retweet the [official Haiku Twitter](https://twitter.com/haikuOS) account posts` -> `You could, for instance, retweet what we post on [our official Twitter account](https://twitter.com/haikuOS).`

That would sound a bit less forced.

File: content/community/getting-involved/promotion/_index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title = "Promotion"
+date = 2021-04-14T15:16:45+10:00
+tags = ["haiku", "promotion", "getting involved", "volunteering"]
++++
+Promotion is a crucial part of ensuring Haiku continues to grow. Without spreading the word, we cannot gain new users and volunteers. 
+
+Whilst Haiku *does* have a [Promotion Team, which you can read more about here](/community/getting-involved/promotion/promotion-team/), you don't need to join the Team to promote the Project, as you will see below.
+
+## What can I do to promote Haiku?
+Letting people know that you use Haiku can help more people discover the Project and decide whether they want to try the OS out. You might want to also mention that our ISOs allow for people to try out the OS without installing it onto a hard disk. You could consider burning the [latest release of Haiku](/get-haiku/) to a USB or CD (make sure the medium is virus-free though) - and pass it around to those around you who may be interested.
```
Comment (panos) [inline content/community/getting-involved/promotion/_index.md:12]: `You could consider burning` -> `You could burn`
`(make sure the medium is virus-free though)` I think this is a bit superfluous and confusing.

File: content/community/getting-involved/promotion/_index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
++++
+type = "article"
+title = "Promotion"
+date = 2021-04-14T15:16:45+10:00
+tags = ["haiku", "promotion", "getting involved", "volunteering"]
++++
+Promotion is a crucial part of ensuring Haiku continues to grow. Without spreading the word, we cannot gain new users and volunteers. 
+
+Whilst Haiku *does* have a [Promotion Team, which you can read more about here](/community/getting-involved/promotion/promotion-team/), you don't need to join the Team to promote the Project, as you will see below.
```
Comment (panos) [inline content/community/getting-involved/promotion/_index.md:9]: This sounds like you're also actively dissuading people from joining the team. The link seems like it's a bit hard to discover to me.

File: content/community/getting-involved/promotion/promotion-team.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+#### Businesses
+Businesses can benefit from Haiku as an operating system, and Haiku has the potential to be a lightweight alternative to the proprietary desktop operating systems businesses currently use. An information sheet for businesses has been drafted, which gives businesses a rundown of Haiku and its advantages and disadvantages as a business-oriented operating system. However, to attract more businesses we must improve Haiku further, and hence we will delay marketing efforts for businesses until Haiku reaches a stable and mature stage. 
+
+#### Partnerships with other Haiku-related sources
+If you haven't heard of them already, [BeSly](https://www.besly.de/index.php/de/) and [Haikunauts](https://haikunauts.home.blog/) are the two major third party Haiku sources. BeSly is a wiki with tips and tricks on using Haiku, whilst Haikunauts is a blog aimed at promoting Haiku as an OS. As mentioned above, we will work in partnership with these two sources and see what we can do together to help spread the word about Haiku.
```
Comment (panos) [inline content/community/getting-involved/promotion/promotion-team.md:64]: `the two major third party Haiku sources` -> `resources about Haiku managed by fans of the project.`

**Comment by jt15s** [comment]
@panos Thanks for those suggestions - I've taken it onboard!

**Pull request merged**